packages/storybook/src/themes/v8/FluentTheme.ts

```diff
@@ -1,11 +1,7 @@
-import { Theme } from '@fluentui/react-theme-provider';
+import { PartialTheme } from '@fluentui/react-theme-provider';
 
-export const FluentTheme: Theme = {
+export const FluentTheme: PartialTheme = {
```

Comment:
we should really consider renaming this package to `@fluentui/storybook-addons` or something that doesn't seem like a fork of storybook.

Reply:
I agree, will take that as a follow-up!